contracts/rust/src/bin/diff_test.rs

```diff
@@ -387,6 +398,53 @@ fn main() {
         Action::TestOnly => {
             println!("args: {:?}", cli.args);
         }
+        Action::GenBLSSig => {
+            let mut rng = jf_utils::test_rng();
+
+            if cli.args.len() != 1 {
+                panic!("Should provide arg1=senderAddress");
+            }
+
+            let sender_address_string = cli.args[0].clone();
+            let sender_address: Address = AbiDecode::decode_hex(sender_address_string).unwrap();
```

Comment:
that's strange, have you tried `ethers::types::Address` instead? (see example [here](https://github.com/gakonst/ethers-rs/blob/c6efa65adc770f98475e5c4e7b644847c20d583e/examples/queries/examples/uniswapv2_pair.rs#L24))

Reply:
This [`FromStr for H160` impl](https://docs.rs/ethers/latest/ethers/core/abi/type.Address.html#impl-FromStr-for-H160) should makes the compiler happy, I don't know why it would err. let's chat.